packages/remix/test/fixtures/07-turborepo/README.md

```diff
@@ -0,0 +1,23 @@
+# Remix Turborepo Vercel
```

Comment:
Let's delete this file

Reply:
c0f161291dbade7a9efc7a33343bb09e922910c1